### ISSUE TITLE
Add options for moving the menu (and preserving Overview and Dash)

### ISSUE THREAD

**Issue by thejan2009** [zacbarton/gnome-shell-extension-bolt#6]
It's an amazing extension, though I wouldn't change it for the functionality of Overview. It'd be great if Bolt menu was hiding in the top right corner, or even down on the left.

**Comment by timorei** [comment]
To be honest, I'd so use this if it didn't kill the overview. It's an awesome extension, but the overview is the cornerstone of the gnome-shell workflow in my experience and I've been working with gnome-shell for months now.

I might actually take a look at the code to find out, how to do this:
- preserve the hotcorner functionality for overview and dash
- keep bolt on the 'windows-key'

If you ever get to make more preferences for this - those things would be really interesting to configure.

**Comment by WG-** [comment]
I totally agree with timorei! Kudos.

**Comment by l300lvl** [comment]
Not to be pesky but I have also thought it would be nice to have the overview still available myself. But, after some thought I am starting to rethink that. Perhaps it would be possible for you to create a similar extension to duplicate some of the overviews functions instead? It would mostly need an ability to retain the dash and work with the dash to dock extension as reported in another bug, as well as include a workspace switcher and expose type open window grid.

It may seem like more of a long way around, but it may prove to be more successful since you can already obviously destroy the overview and create a new work flow of your own.

**Comment by satya164** [comment]
I agree about an option to preserve the hot corner and overview. The overview is a core feature of Gnome Shell and I really don't see any point disabling it. But an option would be nice to keep the overview. Those who want can disable and those who don't want won't disable.

**Comment by xionluhnis** [comment]
Same remark about the overview which seems bad to completely remove (and it provides workspace management already). Why not just assigning different key events ?

I know there are some features which would potentially be redundant (such as the search / recent files), but they can be hidden on both sides.

**Comment by mathematicalcoffee** [comment]
I have a patch that keeps Overview for Windows key/hot corner/button and binds Alt+F1 to Bolt (panel_main_menu keybinding).
I'm going to work on it a little more to try allow Bolt to fire on the windows key (it doesn't work the same as other keybindings), and perhaps preserve hotcorner for overview, but clicking on the 'Bolt' button should bring up Bolt?

**Comment by l300lvl** [comment]
That patch is nice and so many are probably seeking that ability. Hopefully this can be merged upstream and an option can be added to allow choosing hotcorners. I could see having bolt at top left and overview either at top right or bottom left, being very handy.

 For now I wonder the same as you though: is it possible to just have overview as normal hotcorner and bolt open with win key?
